api/sales.js

```diff
@@ -66,12 +70,34 @@ class SalesApi extends Api {
       'batchId', // string or array
       'referenceId', // string
       'noteLike', // string (multiple values requires multiple params)
+      'transactionNum', // integer or array
+      'transactionNumLike', // partial-integer string or array
       'source', // string or array
       'externalKey', // string or array
       'externalKeyLike', // string or array
+      'deferredAccrual', // boolean
       'withWarnings', // boolean
       'status', // string or array
-      'type' // string or array
+      'type' // string or array, values like 'txn', 'event', 'churn'
+    )
+    // TODO check params for possible custom field property names and apply to qs
+    const r = await this.client.get(route, opts)
+    return r && r.body
+  }
+
+  async getSaleTotals (params, opts) {
```

Comment:
could do `async getSaleTotals (params = {}, opts = {})` to use ES6 default params.

Reply:
True, but that only works if the args are `undefined`, whereas my old school implementation handles any falsy value, particularly `null`.

Reply:
That's a great point, I hadn't considered that. Thanks!